presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveBucketedTables.java

```diff
@@ -68,21 +66,18 @@ public Requirement getRequirements(Configuration configuration)
     public void testIgnorePartitionBucketingIfNotBucketed()
     {
         String tableName = mutableTablesState().get(BUCKETED_PARTITIONED_NATION).getNameInDatabase();
-        populateDataToHiveTable(tableName, NATION.getName(), Optional.of("part_key = 'insert_1'"));
-        populateDataToHiveTable(tableName, NATION.getName(), Optional.of("part_key = 'insert_2'"));
+        populateDataToHiveTable(tableName, NATION.getName(), "part_key = 'insert_1'");
+        populateDataToHiveTable(tableName, NATION.getName(), "part_key = 'insert_2'");
 
         testContext().getDependency(QueryExecutor.class, "hive").executeQuery(format("ALTER TABLE %s NOT CLUSTERED", tableName));
 
         assertThat(query(format("SELECT count(*) FROM %s WHERE n_nationkey = 1", tableName)))
                 .containsExactly(row(2));
     }
 
-    private static void populateDataToHiveTable(String destination, String source, Optional<String> partition)
+    private static void populateDataToHiveTable(String destination, String source, String partition)
```

Comment:
Rename to `populateDataToHivePartitionedTable` or `populateHivePartitionedTable` for short.